cookieapp.js

```diff
@@ -0,0 +1,200 @@
+'use strict'
+
+//First store object
+var pikeStore = {
+
+  storeName: '1st and Pike',
+  storeHours: ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'],
+  salesHr: [],
+  minCustomerHr: 23,
+  maxCustomerHr: 65,
+  avgSalesHr: 6.3,
+
+// Calculates random number of customers, which a loop is then used
+//loop runs to calculate number of sales based on random customer output
+  randomCustomerHr: function() {
+    return Math.ceil(Math.random() * (this.maxCustomerHr / this.minCustomerHr));
```

Comment:
This function will potentially return a value much less than either min or max customers.  Take a look at the MDN page for Math.random()

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random